### PR TITLE
Fix missing translation in README_ru

### PR DESCRIPTION
**Pull request by strikoder**
# What does this PR do?
This PR fixes the Russian translation of the README file by translating one line from English to Russian that was forgotten to be translated.

## Before submitting
- [x] This PR improves the docs.

## Fixes #26208 

 @stevhliu 
@MKhalusova

### CODE REVIEW

File: README_ru.md [hunk 1]
```diff
@@ -520,7 +520,8 @@ conda install conda-forge::transformers
...
 1. **[YOSO](https://huggingface.co/docs/transformers/model_doc/yoso)** (from the University of Wisconsin - Madison) released with the paper [You Only Sample (Almost) Once: Linear Cost Self-Attention Via Bernoulli Sampling](https://arxiv.org/abs/2111.09714) by Zhanpeng Zeng, Yunyang Xiong, Sathya N. Ravi, Shailesh Acharya, Glenn Fung, Vikas Singh.
-1. Want to contribute a new model? We have added a **detailed guide and templates** to guide you in the process of adding a new model. You can find them in the [`templates`](./templates) folder of the repository. Be sure to check the [contributing guidelines](./CONTRIBUTING.md) and contact the maintainers or open an issue to collect feedbacks before starting your PR.
+
+1. Хотите внести новую модель? Мы добавили **подробный гид и шаблоны**, чтобы помочь вам в процессе добавления новой модели. Вы можете найти их в папке [`templates`](./templates) репозитория. Обязательно ознакомьтесь с [руководством по внесению вклада](./CONTRIBUTING.md) и свяжитесь с сопровождающими или откройте задачу, чтобы собрать отзывы перед началом работы над вашим пулл-реквестом.
```
Comment (MKhalusova) [inline README_ru.md:524]:
```suggestion
1. Хотите внести новую модель? Мы добавили **подробное руководство и шаблоны**, чтобы помочь вам в процессе добавления новой модели. Вы можете найти их в папке [`templates`](./templates) репозитория. Обязательно ознакомьтесь с [руководством по внесению изменений](./CONTRIBUTING.md) и свяжитесь с ответственным разработчиком или откройте задачу, чтобы собрать отзывы перед началом работы над вашим пулл-реквестом.
```

**Review by MKhalusova — COMMENTED** [review]
Thank you for spotting the missing paragraph, and adding translation. I left a couple of suggestions.

**Comment by Strikoder** [comment]
Okay, tnx Maria!

**Comment by Strikoder** [comment]
Okay, thanks!

**Comment by Strikoder** [comment]
> Thank you for spotting the missing paragraph, and adding translation. I left a couple of suggestions.

Okay, thank you!

**Review by MKhalusova — APPROVED** [review]
LGTM!

**Review by ArthurZucker — APPROVED** [review]

**Pull request merged**